packages/assets-controllers/src/NftController.ts

```diff
@@ -222,41 +218,235 @@ const controllerName = 'NftController';
 /**
  * The external actions available to the {@link NftController}.
  */
-type AllowedActions =
+export type AllowedActions =
   | AddApprovalRequest
   | NetworkControllerGetNetworkClientByIdAction;
 
+export type AllowedEvents =
```

Comment:
Same as with `AllowedActions`, we want this to only be accessible internally (mainly from test files), and not be a package-level export.